### PR TITLE
Countdown

### PR DESCRIPTION
**Pull request by becousae**

### CODE REVIEW

File: site/stylesheets/countdown.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  box-shadow: 0 3px 0 #a1a1a1;
+  color: #242a34;
+  font-family: 'League Gothic', sans-serif;
+  font-size: 4.1em;
+  padding: .6em 0 .5em;
+  text-align: center;
+  width: 1.25em;
+}
+
+.number-sub {
```
Comment (Iepoev) [inline site/stylesheets/countdown.css]: needs dropshadow to fit in with the other text in the header

File: site/javascripts/countdown.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
+/**
+ * WiNA Spotlight Countdown Clock
```
Comment (Iepoev) [inline site/javascripts/countdown.js]: lol

**Comment by Iepoev** [comment]
looking good :+1:

**Comment by Iepoev** [comment]
can this be merged?

**Comment by becousae** [comment]
@Procrat prefers it this way
http://i.imgur.com/Zt2yvFt.png

**Comment by Procrat** [comment]
Looks good to me! :D

File: site/index.html [hunk 3]
```diff
@@ -547,6 +577,8 @@ <h2 class="white"><span class="lines">Contact</span></h2>
   <script src="javascripts/smooth-scroll.js" type="text/javascript"></script>
```
Comment (Silox) [inline site/index.html]: We should use minimized versions for all these javascripts. Preferably from a CDN too if possible.

**Pull request merged**